example/views/getting-started.js

```diff
@@ -87,8 +87,8 @@ ko.applyBindings()
 <h4 class="text-muted">index.html</h4>
 <pre><code data-bind="prism: 'html', noEscape: true">
 ${escape(`<!doctype html>
-<script src="/app.js"></script>
-<app></app>`)}
+<app></app>
+<script src="/app.js"></script>`)}
 </pre></code>
```

Comment:
This breaks IE 6-8, but I'm going to say for documentation I don't think that matters.

http://knockoutjs.com/documentation/component-custom-elements.html#note-custom-elements-and-internet-explorer-6-to-8